packages/core/src/shorthands/gridArea.test.ts

```diff
@@ -0,0 +1,225 @@
+      gridRowEnd: 'header',
+      gridColumnEnd: 'header',
+    });
+  });
+
+  it('for number', () => {
+    expect(gridArea(2)).toEqual({
+      gridRowStart: 2,
+      gridColumnStart: 'auto',
+      gridRowEnd: 'auto',
+      gridColumnEnd: 'auto',
+    });
+  });
+
+  it('for span number', () => {
+    expect(gridArea('span 3')).toEqual({
+      gridRowStart: 'span 3',
+      gridColumnStart: 'auto',
+      gridRowEnd: 'auto',
+      gridColumnEnd: 'auto',
+    });
+  });
+
+  it('for span custom-ident', () => {
+    expect(gridArea('span header')).toEqual({
+      gridRowStart: 'span header',
+      gridColumnStart: 'auto',
+      gridRowEnd: 'auto',
+      gridColumnEnd: 'auto',
+    });
+  });
+});
+
+describe('gridArea(rowStart, columnStart)', () => {
+  it('for auto', () => {
+    expect(gridArea('auto', 'auto')).toEqual({
+      gridRowStart: 'auto',
+      gridColumnStart: 'auto',
+      gridRowEnd: 'auto',
+      gridColumnEnd: 'auto',
+    });
+  });
+
+  it('for custom-ident', () => {
+    expect(gridArea('header', 'footer')).toEqual({
+      gridRowStart: 'header',
+      gridColumnStart: 'footer',
+      gridRowEnd: 'header',
+      gridColumnEnd: 'footer',
+    });
+  });
+
+  it('for number', () => {
+    expect(gridArea(2, 4)).toEqual({
+      gridRowStart: 2,
+      gridColumnStart: 4,
+      gridRowEnd: 'auto',
+      gridColumnEnd: 'auto',
+    });
+  });
+
+  it('for span number', () => {
+    expect(gridArea(3, 'span 2')).toEqual({
+      gridRowStart: 3,
+      gridColumnStart: 'span 2',
+      gridRowEnd: 'auto',
+      gridColumnEnd: 'auto',
+    });
+  });
+
+  it('for span custom-ident', () => {
+    expect(gridArea('header', 'span footer')).toEqual({
+      gridRowStart: 'header',
+      gridColumnStart: 'span footer',
+      gridRowEnd: 'header',
+      gridColumnEnd: 'auto',
+    });
+  });
+});
+
+describe('gridArea(rowStart, columnStart, rowEnd)', () => {
+  it('for auto', () => {
+    expect(gridArea('auto', 'auto', 'auto')).toEqual({
+      gridRowStart: 'auto',
+      gridColumnStart: 'auto',
+      gridRowEnd: 'auto',
+      gridColumnEnd: 'auto',
+    });
+  });
+
+  it('for custom-ident', () => {
+    expect(gridArea(2, 1, 'footer')).toEqual({
+      gridRowStart: 2,
+      gridColumnStart: 1,
+      gridRowEnd: 'footer',
+      gridColumnEnd: 'auto',
+    });
+  });
+
+  it('for number', () => {
+    expect(gridArea(2, 4, 3)).toEqual({
+      gridRowStart: 2,
+      gridColumnStart: 4,
+      gridRowEnd: 3,
+      gridColumnEnd: 'auto',
+    });
+  });
+
+  it('for span number', () => {
+    expect(gridArea('header', 2, 'span 2')).toEqual({
+      gridRowStart: 'header',
+      gridColumnStart: 2,
+      gridRowEnd: 'span 2',
+      gridColumnEnd: 'auto',
+    });
+  });
+
+  it('for span custom-ident', () => {
+    expect(gridArea('header', 'span footer')).toEqual({
+      gridRowStart: 'header',
+      gridColumnStart: 'span footer',
+      gridRowEnd: 'header',
+      gridColumnEnd: 'auto',
+    });
```

Comment:
```suggestion
    expect(gridArea('header', 'span body', 'span footer')).toEqual({
      gridRowStart: 'header',
      gridColumnStart: 'span body',
      gridRowEnd: 'span footer',
      gridColumnEnd: 'auto',
    });
```

Oops, copy-paste issue 😀 This test should have three arguments.